spec/controllers/guides_controller_spec.rb

```diff
@@ -67,4 +67,12 @@
       end
     end
   end
+
+  describe "a malicious user trying to initialise an unwanted Guide STI constant" do
```

Comment:
Do we really need this test? I think it's safe enough because we construct the type in code in the controller #new?